ds-cv-inference/mqtt/mqtt.go

```diff
@@ -61,22 +61,25 @@ func NewMqttConnection(connectionString string) Connection {
 var commandTopicFunction MQTT.MessageHandler = func(client MQTT.Client, msg MQTT.Message) {
 
 	var edgeXMessage map[string]string
```

Comment:
Remove as it should be unused now.

Reply:
nope we need the variable declaration. We add info to the map. I tried to remove it and my IDE got mad at me